src/components/EventForm.jsx

```diff
@@ -0,0 +1,123 @@
+import Radium, { Style } from 'radium';
+import React from 'react';
+
+export default class EventForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      event_start: '',
+      event_end: '',
+      recurs: false,
+    };
+
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleInputChange(event) {
+    const target = event.target;
+    const value = target.type === 'checkbox' ? target.checked : target.value;
+    const name = target.name;
+
+    this.setState({
+      [name]: value,
+    });
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+    fetch('http://localhost:3000/events', {
```

Comment:
Maybe look into this. https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-custom-environment-variables